src/NuGet.Core/NuGet.Common/PathUtil/PathUtility.cs

```diff
@@ -237,30 +237,39 @@ public static string GetRelativePath(string path1, string path2, char separator)
                 ++index;
             }
 
-            var path = "";
-
             // check if path2 ends with a non-directory separator and if path1 has the same non-directory at the end
             if (len1 + 1 == len2 && !string.IsNullOrEmpty(path1Segments[index]) &&
                 string.Equals(path1Segments[index], path2Segments[index], compare))
             {
-                return path;
+                return string.Empty;
```

Comment:
This is a behavior change right?  Does it have any implications?

Reply:
AFAIK, it is not a behavior change. I deleted line 240 `var path = "";` in this PR.

Reply:
Ah sorry I missed that